code/modules/research/rdconsole.dm

```diff
@@ -903,7 +910,7 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 		// if the timer calls this function
 		deltimer(wait_message_timer)
 		wait_message_timer = 0
-	SSnanoui.update_uis(src)
+	SStgui.update_uis(src)
```

Comment:
should I leave this or do `return TRUE` here?

Reply:
If it's part of tgui_act, stick a return TRUE instead of update